### PR TITLE
Feature/e UI 5357

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5357

### Change description ###
Remove Status from Listing Summary
Text change on Create/View Hearing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/hearings/components/listing-information-summary/listing-information-summary.component.html [hunk 1]
```diff
@@ -28,12 +28,6 @@ <h2 class="govuk-heading-m">Listing information summary</h2>
 
 <ng-template #listingContent>
   <dl class="govuk-summary-list govuk-!-margin-bottom-0 govuk-!-padding-right-2">
-    <div class="govuk-summary-list__row" *ngIf="!isCaseStatusListed()">
-      <dt class="govuk-summary-list__key">Status</dt>
-      <dd class="govuk-summary-list__value" [innerHtml]="answerSource.HEARING_RESPONSE_STATUS | transformAnswer:hearingState$ | async">
-      </dd>
-      <dd class="govuk-summary-list__actions"></dd>
-    </div>
```
Comment (rayliangatos) [inline src/hearings/components/listing-information-summary/listing-information-summary.component.html:36]: is this required to remove this status?

File: src/hearings/components/listing-information-summary/listing-information-summary.component.html [hunk 2]
```diff
@@ -107,12 +101,6 @@ <h2 class="govuk-heading-m">Listing information summary</h2>
 
 <ng-template #listingContentMultiDay>
   <dl class="govuk-summary-list govuk-!-margin-bottom-0 govuk-!-padding-right-2">
-    <div class="govuk-summary-list__row" *ngIf="!isCaseStatusListed()">
-      <dt class="govuk-summary-list__key">Status</dt>
-      <dd class="govuk-summary-list__value" [innerHtml]="answerSource.HEARING_RESPONSE_STATUS | transformAnswer:hearingState$ | async">
-      </dd>
-      <dd class="govuk-summary-list__actions"></dd>
-    </div>
```
Comment (rayliangatos) [inline src/hearings/components/listing-information-summary/listing-information-summary.component.html:115]: is this required to remove this status?

File: src/hearings/components/listing-information-summary/listing-information-summary.component.html (hunk 1 above)
Comment (MateiHMCTS) [reply src/hearings/components/listing-information-summary/listing-information-summary.component.html:36]: https://tools.hmcts.net/jira/browse/EUI-5357 doesn't mention 'status' in listing summary' and in screenshot

https://tools.hmcts.net/jira/browse/EUI-6104

File: src/hearings/components/listing-information-summary/listing-information-summary.component.html (hunk 2 above)
Comment (MateiHMCTS) [reply src/hearings/components/listing-information-summary/listing-information-summary.component.html:115]: https://tools.hmcts.net/jira/browse/EUI-5357 doesn't mention 'status' in listing summary' and in screenshot

https://tools.hmcts.net/jira/browse/EUI-6104

**Review by rayliangatos — APPROVED** [review]

File: src/hearings/components/listing-information-summary/listing-information-summary.component.html (hunk 1 above)
Comment (rayliangatos) [inline src/hearings/components/listing-information-summary/listing-information-summary.component.html:36]: Adeel said this should be retained, we need him to correct the testing ticket.

File: src/hearings/components/listing-information-summary/listing-information-summary.component.html [hunk 3]
```diff
@@ -107,12 +101,6 @@ <h2 class="govuk-heading-m">Listing information summary</h2>
 
 <ng-template #listingContentMultiDay>
   <dl class="govuk-summary-list govuk-!-margin-bottom-0 govuk-!-padding-right-2">
-    <div class="govuk-summary-list__row" *ngIf="!isCaseStatusListed()">
```
Comment (rayliangatos) [inline src/hearings/components/listing-information-summary/listing-information-summary.component.html:110]: Adeel said this should be retained, we need him to correct the testing ticket.

Comment (MateiHMCTS) [reply src/hearings/components/listing-information-summary/listing-information-summary.component.html]: I have added Status back.
Do we still want to go ahead with the change 'party' to 'participant' as per the ticket? or should I revert this completely?

**Pull request merged**